app/models/distribution.rb

```diff
@@ -3,7 +3,10 @@
 
 class Distribution < BaseModel
   belongs_to :distributable, polymorphic: true, touch: true
+  belongs_to :audio_version_template
```

Comment:
I may be wrong though, as on the b'cast side we do distribute via subauto multiple versions, like the single file version and a segmented version. So depending on how that gets implemented when we bring in b'cast, this could change. Currently that is modeled by having multiple templates associated to a single series via a series_id fk on the template table. I believe this could shift to associating the template to the form of distribution as well.

So that gets messy, it means a distro could have multiple templates. Likewise, I wonder if a template might make sense in multiple distros - that is fine with the current setup, but falls apart if we shift the fk to associate the template to the distro_id, at that point, I think we many need a join table :^/